benchexec/tools/verifast.py

```diff
@@ -0,0 +1,41 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2023 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.result as result
+import benchexec.tools.template
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info module for VeriFast (https://github.com/verifast/verifast)
+    """
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("verifast", subdir="bin")
+
+    def name(self):
+        return "VeriFast"
+
+    def version(self, executable):
+        return self._version_from_tool(executable)
+
+    def determine_result(self, run):
+        for line in run.output:
+            line = line.strip()
+            if line.startswith("0 errors found"):
```

Comment:
It seems this would also match something like `10 errors found`. Is it possible to strengthen this check, e.g., by avoiding `startswith`? It might also be a good idea to strengthen the other checks where possible.

Reply:
Hm, not sure what I thought here. Of course it does not match `10 errors found` *due to* the `startswidth`. I think I misread as `endswith`. Sorry.